### PR TITLE
Server postgres

### PR DESCRIPTION
**Pull request by Gabriellji**
- Connect Postgresql
- Create CRUD operations for product entity
- Create model for product

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require("dotenv").config();
```
Comment (leovvay) [inline src/app.js]: move to entry pooint(server.js)

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const productRouter = require("./routes/product.route");
+
+const app = express();
+
+db.connector
+  .sync()
+  .then(() => console.log("find current DB"))
+  .catch((err) => console.error(`sync faild: ${err}`));
+
+app.use(express.urlencoded({ extended: false }));
```
Comment (leovvay) [inline src/app.js:12]: what is it

File: src/db/db.client.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require('dotenv').config()
```
Comment (leovvay) [inline src/db/db.client.js]: rm

File: src/db/db.client.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        await connector.authenticate()
+        console.log('We are Connected!')
+    } catch(err) {
+        console.log(`Something failed on the DB connection: ${err}`)
+    }
+}
+
+authenticate(connector)
+
+const db = {}
```
Comment (leovvay) [inline src/db/db.client.js]:
```suggestion
const db = {
Sequelize,
connector,
product: require('../models/product.model')(Sequelize, connector)
}
```

File: src/routes/product.route.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../db/db.client");
+const Product = db.product;
```
Comment (leovvay) [inline src/routes/product.route.js]:
```suggestion
const {product:Product} = db;
```

File: src/routes/product.route.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+// Public
+// GET all
+router.get("/", async (req, res) => {
+  try {
+    const products = await Product.findAll();
+    res.json(products);
+  } catch (err) {
+    console.error(err.message);
+    res.status(500).send("Server Error");
```
Comment (leovvay) [inline src/routes/product.route.js]: move to constant file http_codes 
constants/httpCodes.js
export const SERVER_ERROR = 500;

const {SERVER_ERROR} = require('const/httpcodes');

Comment (leovvay) [reply src/routes/product.route.js]: * as httpcodes;

Comment (leovvay) [reply src/routes/product.route.js]: mvoe to error messages

**Review by leovvay — APPROVED** [review]

File: src/app.js (hunk 1 above)
Comment (Gabriellji) [reply src/app.js]: done

File: src/db/db.client.js (hunk 3 above)
Comment (Gabriellji) [reply src/db/db.client.js]: done

File: src/db/db.client.js (hunk 4 above)
Comment (Gabriellji) [reply src/db/db.client.js]: done

File: src/routes/product.route.js (hunk 5 above)
Comment (Gabriellji) [reply src/routes/product.route.js]: done

File: src/routes/product.route.js (hunk 6 above)
Comment (Gabriellji) [reply src/routes/product.route.js]: done

**Pull request merged**